config2.json

```diff
@@ -0,0 +1,84 @@
+{
+    "max_open_trades": 3,
+    "stake_currency": "USDT",
+    "stake_amount": 50,
+    "tradable_balance_ratio": 0.99,
+    "fiat_display_currency": "USD",
+    "dry_run": true,
+    "cancel_open_orders_on_exit": false,
+    "unfilledtimeout": {
+        "buy": 10,
+        "sell": 30,
+        "unit": "minutes"
+    },
+    "bid_strategy": {
+        "price_side": "bid",
+        "ask_last_balance": 0.0,
+        "use_order_book": true,
+        "order_book_top": 1,
+        "check_depth_of_market": {
+            "enabled": false,
+            "bids_to_ask_delta": 1
+        }
+    },
+    "ask_strategy": {
+        "price_side": "ask",
+        "use_order_book": true,
+        "order_book_top": 1
+    },
+    "exchange": {
+        "name": "binance",
+        "key": "r7jNtIb7EWp8VR0Y9wyONEboIDYJ1P6LtY7ywpIH3knxDXQDukeAPTLMztlP2OGB",
+        "secret": "qdtxxwCA5USIb1F3086C3a0RlTqoRSX8pzxWqjmTDF8ap5v6CKLNn2GlbukiADHJ",
```

Comment:
@tef-github 
You better rotate / remove these keys from the exchnage immediately. 
They've now been exposed and must therefore be assumed as compromised.

The same is true for the telegram bot keys - although you can't rotate them but must remove the bot and create a new one.

Reply:
Thank you. I have removed both.